### ISSUE TITLE
Purpose of the project

### ISSUE THREAD

**Issue by CosminPerRam** [Beeamo/MyAnimeList-FriendBot#7]
Hey, hope you don't mind me getting through GitHub issues, but what exactly is the purpose of this project?

Don't get me wrong, it's a cool rewrite that also seems to bring some more features to the table, but for what exactly? Just for mass friend request spamming?

**Comment by Beeamo** [comment]
Yeah it's really for better friend request spamming in simplest terms. It initially got started because someone wanted my code from when I actually used this but in the last 8 months or so the program stopped working in more ways then one. It started very similar to the old project but since the my project had it's issues and wasn't very friendly for other people to use I decided to give it a few updates in the last week to get it working again and making it easier to use.

The changes from the initial repository were really to make it less annoying for the individual user of mal. You'll only receive 1 friend request that you choose to deny instead of multiple from the same person.
